### PR TITLE
Updated documentation to reflect new OMF plugin

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: docs/quick_start.rst [hunk 1]
```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
...
+
+- Basic Information
+   - **Endpoint:** Select what you wish to connect to, in this case PI Web API.
+   - **Server hostname:** The hostname or address of the PI Server.
+   - **Server port:** The port the PI Web API OMF endpoint is listening on. Leave as 0 if you are using the default port.
+   - **Data Source:** Defines which data is sent to the PI Server. The readings or Fledge's internal statistics.
+   - **Static Data:** Data to include in every reading sent to PI.  For example, you can use this to specify the location of the devices being monitored by the Fledge server.
+- Asset Framework
+   - **Asset Framework Hierarchies Tree:** The location in the asset framework into which the data will be inserted. All data will be inserted at this point in the asset framework unless a later rule overrides this.
+   - **Asset Framework Hierarchies Rules:** A set of rules that allow specific readings to be placed elewhere in the asset freamework. These rules can be based on the name of the asset itself or some metadata associated with the asset.
```
Comment (ashish-jabble) [inline docs/quick_start.rst:284]: typo's - elsewhere, framework

File: docs/quick_start.rst [hunk 2]
```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
...
+   - **HTTP Timeout:** Number of seconds to wait before Fledge will time out an HTTP connection attempt.
+- Other (Rarely changed)
+   - **Integer Format:** Used to match Fledge data types to the data type configured in PI. This defaults to int64 but may be set to any OMF data type compatible with integer data, e.g. int32.
+   - **Number Format:** Used to match Fledge data types to the data type configured in PI. The defaults is float64 but may be set to any OMF datatype that supports floating point values.
+   - **Compression:** Compress the readings data before sending it to the PI System.
+
+PI Web API OMF Endpoint
+-----------------------
+
+To use the PI Web API OMF endpint first  ensure the OMF option was included in your PI Server when it was installed.
```
Comment (ashish-jabble) [inline docs/quick_start.rst:265]: typo - endpoint

File: docs/quick_start.rst [hunk 3]
```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
...
+   - **Server port:** The port the PI Web API OMF endpoint is listening on. Leave as 0 if you are using the default port.
+   - **Data Source:** Defines which data is sent to the PI Server. The readings or Fledge's internal statistics.
+   - **Static Data:** Data to include in every reading sent to PI.  For example, you can use this to specify the location of the devices being monitored by the Fledge server.
+- Asset Framework
+   - **Asset Framework Hierarchies Tree:** The location in the asset framework into which the data will be inserted. All data will be inserted at this point in the asset framework unless a later rule overrides this.
+   - **Asset Framework Hierarchies Rules:** A set of rules that allow specific readings to be placed elewhere in the asset freamework. These rules can be based on the name of the asset itself or some metadata associated with the asset.
+- PI Web API authentication
+   - **PI Web API Authentication Method:** The authentication method to be used, anonymous equates to no authentication, basic authenetication requires a user name and password and Kerberos allows integration with your single sign on environment.
```
Comment (ashish-jabble) [inline docs/quick_start.rst:286]: ~authenetication~ authentication

File: docs/quick_start.rst [hunk 4]
```diff
@@ -269,9 +372,11 @@ Troubleshooting and Support Information
 | |support| |
 +-----------+
 
-Fledge keep detailed logs of system events for both auditing and troubleshooting use.  To access them, click "Logs" in the left menu bar.  There are three logs in the system:
+Fledge keep detailed logs of system events for both auditing and troubleshooting use.  To access them, click "Logs" in the left menu bar.  There are five logs in the system:
 
   - **Audit:** Tracks all configuration changes and data uploads performed on the Fledge system.
+  - **Notifications:** If you are using the Fledge notifcation service this log will give details of notifications that have been triggered
```
Comment (ashish-jabble) [inline docs/quick_start.rst:378]: typo - notification

File: docs/quick_start.rst [hunk 5]
```diff
@@ -207,10 +208,15 @@ Enabling and Disabling Data Destinations
...
+Using the Fledge OMF plugin
 ###########################
 
-OSISoft PI systems are one of the most common destinations for Fledge data.  To send data to a PI server, open and sign into the PI Relay Data Connection Manager.
+OSISoft data historians are one of the most common destinations for Fledge data.  Fledge supports the full range of OSISoft historians; the PI System, Edge Data Store (EDS) and OSISoft Cloud Services (OCS). To send data to a PI server you may use either the older PI Connector Relay or the newer PI Web API OMF endpoint It is recommended that new users use the PI Web API OMF endpoint rather then the Connector Relay.
+
+PI Connector Relay
+------------------
+
+To use the Reconnector Relay, open and sign into the PI Relay Data Connection Manager.
```
Comment (ashish-jabble) [inline docs/quick_start.rst:219]: Reconnector or connector?

File: docs/quick_start.rst [hunk 6]
```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
...
+PI Web API OMF Endpoint
+-----------------------
+
+To use the PI Web API OMF endpint first  ensure the OMF option was included in your PI Server when it was installed.  
+
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
+The second screen will request the following information:
+
++---------------------+
+| |omf_plugin_config| |
```
Comment (ashish-jabble) [inline docs/quick_start.rst:271]: This image has with endpoint `OSI Cloud Services` selected. It should be with `PI Web API`

File: docs/quick_start.rst [hunk 7]
```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
 
 Finally, select the new Fledge application. Click "More" at the bottom of the Configuration panel. Make note of the Producer Token and Relay Ingress URL.
 
-Now go to the Fledge user interface, create a new North instance and select the “pi_server” plugin on the first screen.
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
+The second screen will request the following information:
+
++---------------------+
+| |omf_plugin_config| |
```
Comment (ashish-jabble) [inline docs/quick_start.rst:243]: This image has with endpoint `OSI Cloud Services` selected. It should be with `Connector Relay`

File: docs/quick_start.rst [hunk 8]
```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
...
+To use the OSISoft Edge Data Store first install Edge Data Store on the same machine as your Fledge instance. It is a limitation of Edge Data Store that it must reside on the same host as any system that connects to it with OMF.
+
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
 The second screen will request the following information:
 
-+--------------------+
-| |pi_plugin_config| |
-+--------------------+
++---------------------+
+| |omf_plugin_config| |
```
Comment (ashish-jabble) [inline docs/quick_start.rst:308]: This image has with endpoint `OSI Cloud Services` selected. It should be with `Edge Data Store`

**Comment by MarkRiddoch** [comment]
@ashish-jabble The download link will ave to be dealt with separately we currently do not have the landing page on the website and there is no Fledge release, so until these happen it can not be fixed.

File: docs/quick_start.rst [hunk 9]
```diff
@@ -207,10 +210,15 @@ Enabling and Disabling Data Destinations
 
 To enable or disable a data source, click on its name in the North Services screen. Under the list of data source parameters, there is a check box to enable or disable the service.  If you make any changes, click on the “save” button in the bottom panel near the check box to save the new configuration.
 
-Using the Fledge PI plugin
+Using the Fledge OMF plugin
 ###########################
 
-OSISoft PI systems are one of the most common destinations for Fledge data.  To send data to a PI server, open and sign into the PI Relay Data Connection Manager.
+OSISoft data historians are one of the most common destinations for Fledge data.  Fledge supports the full range of OSISoft historians; the PI System, Edge Data Store (EDS) and OSISoft Cloud Services (OCS). To send data to a PI server you may use either the older PI Connector Relay or the newer PI Web API OMF endpoint It is recommended that new users use the PI Web API OMF endpoint rather then the Connector Relay.
```
Comment (sstefano69) [inline docs/quick_start.rst:216]: PI Web API OMF endpoint. It is recommended

**Comment by ashish-jabble** [comment]
@MarkRiddoch Would you mind to fix more GitHub links for docs related itself here or another JIRA?
```
1) fledge/docs
91_version_history.rst
   <a href="https://github.com/fledge/Fledge/blob/1.1/python/requirements.txt" target="_blank">check here</a>
  - **coap**: coap south plugin was moved into its own repository https://github.com/fledge/fledge-south-coap
  - **http**: http south plugin was moved into its own repository https://github.com/fledge/fledge-south-http
92_downloads.rst
   <a href="https://github.com/fledge/storage-postgres" target="_blank">https://github.com/fledge/storage-postgres</a>
We don't have this repo on fledge, so can be deleted from here
quick_start.rst
- As source code from https://github.com/fledge/.  Instructions for downloading and building Fledge source code can be found in the Fledge Developer’s Guide
REDHAT.rst
.. _fledge-pkg: https://github.com/fledge/fledge-pkg

2) fledge/docs/building_fledge
01_introduction.rst
.. _Fledge project on GitHub: https://github.com/fledge/Fledge/issues
04_installation.rst
   <a href="https://github.com/fledge/fledge-snap/issues" target="_blank">GitHub issues database</a>
We don't have this repo on fledge, so can be deleted from here
04_utilities.rst
   <a href="https://github.com/fledge/Fledge/issues" target="_blank">GitHub</a>
building_fledge.rst
   <a href="https://github.com/fledge/Fledge" target="_blank">https://github.com/fledge/Fledge</a>
First, clone the Github repository with the usual command: |br| ``git clone https://github.com/fledge/Fledge.git`` |br| The project should have been added to your machine under the *Fledge* directory.
05_testing.rst
Note: This following instructions assume you have downloaded and installed the CoAP south plugin from https://github.com/fledge/fledge-south-coap.
  $ git clone https://github.com/fledge/fledge-south-coap

3) fledge/docs/plugin_developers_guide
03_01_DHT11.rst
   <a href="https://github.com/fledge/fledge-south-dht11" target="_blank">Fledge DHT11 South Plugin</a>
```

**Comment by MarkRiddoch** [comment]
> @MarkRiddoch Would you mind to fix more GitHub links for docs related itself here or another JIRA?
> 
> ```
> 1) fledge/docs
> 91_version_history.rst
>    <a href="https://github.com/fledge/Fledge/blob/1.1/python/requirements.txt" target="_blank">check here</a>
>   - **coap**: coap south plugin was moved into its own repository https://github.com/fledge/fledge-south-coap
>   - **http**: http south plugin was moved into its own repository https://github.com/fledge/fledge-south-http
> 92_downloads.rst
>    <a href="https://github.com/fledge/storage-postgres" target="_blank">https://github.com/fledge/storage-postgres</a>
> We don't have this repo on fledge, so can be deleted from here
> quick_start.rst
> - As source code from https://github.com/fledge/.  Instructions for downloading and building Fledge source code can be found in the Fledge Developer’s Guide
> REDHAT.rst
> .. _fledge-pkg: https://github.com/fledge/fledge-pkg
> 
> 2) fledge/docs/building_fledge
> 01_introduction.rst
> .. _Fledge project on GitHub: https://github.com/fledge/Fledge/issues
> 04_installation.rst
>    <a href="https://github.com/fledge/fledge-snap/issues" target="_blank">GitHub issues database</a>
> We don't have this repo on fledge, so can be deleted from here
> 04_utilities.rst
>    <a href="https://github.com/fledge/Fledge/issues" target="_blank">GitHub</a>
> building_fledge.rst
>    <a href="https://github.com/fledge/Fledge" target="_blank">https://github.com/fledge/Fledge</a>
> First, clone the Github repository with the usual command: |br| ``git clone https://github.com/fledge/Fledge.git`` |br| The project should have been added to your machine under the *Fledge* directory.
> 05_testing.rst
> Note: This following instructions assume you have downloaded and installed the CoAP south plugin from https://github.com/fledge/fledge-south-coap.
>   $ git clone https://github.com/fledge/fledge-south-coap
> 
> 3) fledge/docs/plugin_developers_guide
> 03_01_DHT11.rst
>    <a href="https://github.com/fledge/fledge-south-dht11" target="_blank">Fledge DHT11 South Plugin</a>
> ```

Sure. We should update the white labelling script to cope with this, foglamp-iot has no meaning, but equally we don't want to point at the old foglamp repository, it should still point to the fledge repository.

File: docs/building_fledge/04_installation.rst [hunk 10]
```diff
@@ -24,7 +24,7 @@
 
 .. |fledge-snap issues| raw:: html
 
-   <a href="https://github.com/fledge/fledge-snap/issues" target="_blank">GitHub issues database</a>
+   <a href="https://github.com/fledge-iot/fledge-snap/issues" target="_blank">GitHub issues database</a>
```
Comment (ashish-jabble) [inline docs/building_fledge/04_installation.rst:27]: There is no repo exists

File: docs/92_downloads.rst [hunk 11]
```diff
@@ -25,7 +25,7 @@
 
 .. |postgres for fledge| raw:: html
 
-   <a href="https://github.com/fledge/storage-postgres" target="_blank">https://github.com/fledge/storage-postgres</a>
+   <a href="https://github.com/fledge-iot/storage-postgres" target="_blank">https://github.com/fledge-iot/storage-postgres</a>
```
Comment (ashish-jabble) [inline docs/92_downloads.rst:28]: There is no repo exists

File: docs/building_fledge/04_utilities.rst [hunk 12]
```diff
@@ -14,7 +14,7 @@
 
 .. |GH Issues| raw:: html
 
-   <a href="https://github.com/fledge/Fledge/issues" target="_blank">GitHub</a>
+   <a href="https://github.com/fledge-iot/Fledge/issues" target="_blank">GitHub</a>
```
Comment (ashish-jabble) [inline docs/building_fledge/04_utilities.rst:17]: fyi, Fledge issues option is NOT available on GitHub for any user. May be they have disabled or turn issues off for the repository

**Pull request merged**